src/sequences/defaultRelease.js

```diff
@@ -45,14 +55,8 @@ export const defaultReleaseSequence = [
   cook.createReleaseNotes('avatar'),
   cook.byBranch,
   {
-    master: [
-      cook.tagCurrentCommit,
-      cook.pushTagToRemote,
-      cook.byReleaseTarget,
-      {
-        github: cook.createGithubRelease,
-      },
-    ],
+    master: publishToGithub,
```

Comment:
@henri-hulski This is the only place where we need "master" for the rest we use tags to detect the last release.